schemas/stsci.edu/asdf/0.1.0/core/history_entry.yaml

```diff
@@ -0,0 +1,28 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://stsci.edu/schemas/asdf/0.1.0/core/history_entry"
+title: |
+  An entry in the file history.
+
+tag: "tag:stsci.edu:asdf/0.1.0/core/asdf"
+type: object
+properties:
+  description:
+    description: |
+      A description of the transformation performed.
+    type: string
+
+  time:
+    description: |
+      A timestamp for the operation.
+    type: string
+    format: date-time
+
+  software:
+    description: |
+      A description of the software that performed the operation.
```

Comment:
Although now I have a question about this--is this the ASDF library that produced the file, or is this meant to be some other software that assembled or modified the file (regardless of what ASDF library it's using)?  I assume the latter, which makes sense.  But I wasn't thinking clearly about that at first.  So in that case `asdf_library` would just be the *last* ASDF library that updated the file?